tap_shopify/schemas/products.json

```diff
@@ -303,6 +303,24 @@
               "null",
               "string"
             ]
+          },
+          "presentment_prices": {
+            "type": ["null", "array"],
+            "items": {
+              "properties": {
+                "price": {
```

Comment:
This schema is wrong, I think we are expecting data like:
```
"presentment_prices": [{"price": "1700.0", "currency_code": "EUR"}, {"price": "1210.0", "currency_code": "GBP"}, {"price": "2170.0", "currency_code": "USD"}]
```

Why do we have it structured as `price.amount` and `price.currency_code`?